recipes/default.rb

```diff
@@ -30,3 +30,21 @@
 
   node['zookeeper-cluster']['service'].each_pair { |k, v| r.send(k, v) }
 end
+
+config_directory = node.default['zookeeper-cluster']['service']['environment']['ZOOCFGDIR']
+
+template "#{config_directory}/log4j.properties" do
```

Comment:
This should be added [exactly how was done in the Kafka Config provider](https://github.com/bloomberg/kafka-cookbook/blob/master/libraries/kafka_config.rb).